test/extensions/filters/http/compressor/gzip_compressor_filter_test.cc

```diff
@@ -0,0 +1,170 @@
+#include <memory>
+
+#include "common/common/hex.h"
+
+#include "extensions/compression/gzip/compressor/config.h"
+#include "extensions/compression/gzip/decompressor/zlib_decompressor_impl.h"
+#include "extensions/filters/http/compressor/compressor_filter.h"
+
+#include "test/mocks/http/mocks.h"
+#include "test/mocks/stats/mocks.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Compressor {
+
+class GzipCompressorFilterTest : public testing::Test {
```

Comment:
I think that as long as we have good unit tests for the generic compressor filter and good unit tests for the gzip compressor, then we don't need unit tests for the gzip filter.